web/packages/teleport/src/Resources/ResourceCard.tsx

```diff
@@ -0,0 +1,164 @@
+/**
+ * Copyright 2023 Gravitational, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import styled from 'styled-components';
+
+import { Box, ButtonBorder, Flex, Label, Text } from 'design';
+
+import { CheckboxInput } from 'design/Checkbox';
+import { ResourceIcon, ResourceIconName } from 'design/ResourceIcon';
+import {
+  ApplicationsIcon,
+  DatabasesIcon,
+  DesktopsIcon,
+  KubernetesIcon,
+  ServersIcon,
+} from 'design/SVGIcon';
+
+import { AgentKind, UnifiedResourceKind } from 'teleport/services/agents';
+
+const SingleLineBox = styled(Box)`
+  overflow: hidden;
+  white-space: nowrap;
+`;
+
+type Props = {
+  resource: AgentKind;
+};
+export const ResourceCard = ({ resource }: Props) => {
+  const name = agentName(resource);
```

Comment:
lets deprecate the use of `agent` to mean resources? because people use the term resources anyways (it's more accurate) and this file is called resource.

referring the resource type `db` for example as agent is incorrect as well (an example of an agent is type [db_service](https://github.com/gravitational/teleport/blob/master/api/constants/constants.go#L112))  

we also renamed `agent` to `service` so you'll see those two to mean the same thing


history: we used the term `agent` to help distinguish between (nodes, apps, kube...etc) from [these resources](https://github.com/gravitational/teleport/blob/master/web/packages/teleport/src/services/resources/resource.ts), you'll see in the code base we have lots of things called resources b/c everything is a resource, `db_service` is a resource

i know we have types called `Agent`, perhaps we can rename those to `UnifiedResource` now (and unified resource will refer to nodes, apps, kube ...etc)? 

idk... naming is so hard ;;

Reply:
@kimlisa , thanks a lot for your explanations, this really clears the fog around these concepts. I will rename the references to agent concept in my code.

Reply:
Side note: I also really don't like the `AgentKind` name (That's not a kind, that's an agent! Or, actually, a resource). So renaming it to `UnifiedResource` indeed sounds like a good idea.